seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -33,20 +33,19 @@
       "id": "bd7123c8c441eddfaeb5bdef",
       "title": "Say Hello to HTML Elements",
       "description": [
-        "Welcome to freeCodeCamp's coding challenges. These will walk you through web development step-by-step.",
-        "We recommend you do these in order, but feel free to skip around by clicking the map on the left.",
-        "Also, if you want to save your progress, you should join our community by creating an account. In just a few minutes, you can join our chat room, forum, and even a local study group in your city.",
-        "First, you'll start by building a simple web page using HTML. You can edit <code>code</code> in your <code>text editor</code>, which is embedded into this web page.",
-        "Do you see the code in your text editor that says <code>&#60;h1&#62;Hello&#60;/h1&#62;</code>? That's an HTML <code>element</code>.",
-        "Most HTML elements have an <code>opening tag</code> and a <code>closing tag</code>.",
+        "Welcome to freeCodeCamp's coding challenges. These will walk you through web development, step by step.",
+        "We recommend you do challenges in order but feel free to jump around using the map in the menu.",
+        "To save progress, create an account and join our community. In minutes, you can join our chat room, forum and even a study group in your city.",
```

Comment:
I think we can find a happy medium between the original and edited version of this paragraph to make it sound both personal and clear:
"If you want to save your progress, you should create an account and join our community. In minutes, ..."